README.md

```diff
@@ -720,6 +722,23 @@ containing the `Animal`'s name and color in columns named `animal_name` and `col
 #### Constraints and Rules
 - Must be on a property field that is not of list type.
 
+### not_in_collection
+#### Example Use
+```graphql
+{
+    Animal {
+        name @output(out_name: "animal_name")
+        color @output(out_name: "color")
+              @filter(op_name: "not_in_collection", value: ["$colors"])
+    }
+}
+```
+This returns one row for every `Animal` which has a color not contained in a list of colors,
```

Comment:
Can you change this block so that it follows the format of surrounding blocks? In other words, split this into two sentences and have the second start with "Each row...".

Reply:
Seems like some blocks have this format and others have the "Each row" format. I'll go with the latter for clarity.